src/utils.ts

```diff
@@ -391,3 +395,40 @@ export const extractOverrides = (node: FullFunctionDefinition): string | null =>
 
   return `(${Array.from(contractsSet.contracts).join(', ')})`;
 };
+
+/**
+ * Extracts if there are nested mappings in a struct
+ * @dev This function is recursive, loops through all the fields of the struct and nested structs
+ * @param node The struct to extract the mappings from
+ * @returns True if there are nested mappings, false otherwise
+ */
+export const extractStructsNestedMappings = (node: TypeName): boolean => {
```

Comment:
I'd shorten the name of the function a bit and hinted that it returns a boolean. Right now it can be understood as a function returning some nested mappings. Perhaps `hasNestedMappings` or something like that would work, wdyt?